tests/postgres_tests/test_array.py

```diff
@@ -173,12 +173,32 @@ def test_in(self):
             self.objs[:2]
         )
 
+    @unittest.expectedFailure
```

Comment:
Please add comments explaining the rationale. (see https://code.djangoproject.com/ticket/23943 for past confusion about the purpose of `expectedFailure`)